cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -159,6 +153,18 @@ func (o *ReleaseOptions) releaseResources(appSpec kcv1alpha1.AppSpec, pkgBuild c
 	return nil
 }
 
+func generateValuesSchema(pkgBuild cmdpkgbuild.PackageBuild) (*kcdatav1alpha1.ValuesSchema, error) {
+	var yttPaths []string
+	for _, templateStage := range pkgBuild.Spec.Template.Spec.App.Spec.Template {
+		if templateStage.HelmTemplate != nil {
```

Comment:
This logic relies on the user always using HelmTemplate first if they do, which is supposed to be the case

Not sure if we should make it a Bir generic

Reply:
Not able to comprehend. You want to do something like this:

``` 
for index, templateStage := range pkgBuild.Spec.Template.Spec.App.Spec.Template {
		
               if templateStage.HelmTemplate != nil && index == 0 {

                ...
}```